pelican/plugins/seo/seo_report/__init__.py

```diff
@@ -85,13 +85,13 @@ def _page_title_report(self, page_title_analysis):
                 > SEOReport.PAGE_TITLE_RECOMMENDED_LENGTH[-1]
             ):
                 report["content"]["to_improve"].append(
-                    "Your title is too long. The maximum recommended \
-                        length is 70 characters."
+                    "Your title is too long. The maximum recommended "
+                    "length is 70 characters."
                 )
 
         else:
             report["content"]["problems"].append(
-                "Title is missing. Create one to improve your SEO."
+                "The title is missing. Create one to improve your SEO."
```

Comment:
I'm not 100% sure this is correct :sweat_smile: os it would be good if someone could confirm it.

Reply:
Is ok